appveyor.yml

```diff
@@ -0,0 +1,22 @@
+# Test against the latest version of this Node.js version
+environment:
+  nodejs_version: "8.11.3"
+
+# Install scripts. (runs after repo cloning)
+install:
+  # Get the latest stable version of Node.js or io.js
+  - ps: Install-Product node $env:nodejs_version
+  # install modules
+  - yarn install
+
+# Post-install test scripts.
+test_script:
+  # Output useful info for debugging.
+  - node --version

```

Comment:
don't put the node version and the yarn version on the test-script. 
The node version is defined explicitly above and yarn is printing the version in every use.